src/main/java/com/icthh/xm/ms/entity/service/XmEntitySpecService.java

```diff
@@ -49,28 +43,74 @@
 public class XmEntitySpecService implements RefreshableConfiguration {
```

Comment:
what are you doing with file?
why full file was changed? and git history broken?

Reply:
because it is reorganized and refactored